homeassistant/components/sensor/google_travel_time.py

```diff
@@ -108,10 +152,24 @@ def unit_of_measurement(self):
     @Throttle(MIN_TIME_BETWEEN_UPDATES)
     def update(self):
         """Get the latest data from Google."""
-        now = datetime.now()
+        options_copy = self._options.copy()
+        dtime = options_copy.get('departure_time')
+        atime = options_copy.get('arrival_time')
+        if dtime is not None and ':' in dtime:
+            options_copy['departure_time'] = convert_time_to_utc(dtime)
+
+        if atime is not None and ':' in atime:
+            options_copy['arrival_time'] = convert_time_to_utc(atime)
+
+        departure_time = options_copy.get('departure_time')
+        arrival_time = options_copy.get('arrival_time')
+        if departure_time is not None and arrival_time is not None:
+            wstr = ("Google Travel Time: You can not provide both arrival "
```

Comment:
This can be caught using validation 👍  https://github.com/alecthomas/voluptuous/blob/master/voluptuous/schema_builder.py#L842